arkane/gaussian.py

```diff
@@ -33,16 +28,23 @@
 #                                                                             #
 ###############################################################################
 
-import math
-import numpy
+"""
+Arkane Gaussian module
+Used to parse Gaussian output files
+"""
+
 import logging
+import math
 import os.path
 
+import numpy as np
+
 import rmgpy.constants as constants
-from rmgpy.statmech import IdealGasTranslation, NonlinearRotor, LinearRotor, HarmonicOscillator, Conformer
 from rmgpy.exceptions import InputError
```

Comment:
I don't think this import statement is needed anymore, get rid of it during the rebase